portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,56 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random Bible verse to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+let bibleVerses = new Map();
+bibleVerses.set('Romans 5: 3-5', 'Not only so, but we also glory in our ' +
+		'sufferings, because we know that suffering produces perseverance; ' +
```

Comment:
Follow the 4-space multi-line wrapping indentation rule, as here - https://google.github.io/styleguide/jsguide.html#features-strings-no-line-continuations

Reply:
Same problem as before. I did follow indentation rules. How can I fix? x2

Reply:
I think I might've fixed all these indentation problems with Gloria's suggestion of switching from tabs to spaces.